src/TrackGenerator.cpp

```diff
@@ -559,7 +605,9 @@ void TrackGenerator::generateTracks() {
     }
   }
 
+  /* Initialize the track boundary conditions and set the track UIDs */
```

Comment:
Also I would say "UID" as opposed to "uid" in comments to be more clear as you do here